transforms/language/doc_quality/python/README.md

```diff
@@ -16,7 +16,21 @@ In this transform, following metrics will be included:
   - > 30% ending with an ellipsis.
   - Require that 80% of words in a document contain at least one alphabetic character, and apply a "stop word" filter, to remove documents that do NOT contain at least TWO of the following English words: the, be, to, of, and, that, have, with; this adequately deals with ostensibly English documents that contain no coherent English text.
 
-
+| output column name | data type | description | supported language |
```

Comment:
This is great.  Although the paragraph above it seems to conflict somewhat.  Or is that paragraph more of an example?  If so, maybe it should use the work "example" somewhere?